### ISSUE TITLE
change location of saved videos

### ISSUE THREAD

**Issue by melippmann** [ChicoState/open-source-security-camera#39]
**Is your feature request related to a problem? Please describe.**
recorded videos are currently saved to the project folder but should be saved elsewhere

**Describe the solution you'd like**
Users should be able to choose where recorded videos are saved on the device/storage drive

**Comment by kevdouglass** [comment]
I see that this issue (#39) has 2 people assigned @DJ-IRL and @melippmann , however I'm pretty sure this is the one you needed my help with? Just want to make sure this is the "Change Path" issue, correct?

**Comment by melippmann** [comment]
@kevdouglass correct, originally DJ-IRL was going to help with this, but other things took priority and then you and I made some good progress on it, so I am going to re-assign you to the task and take him off it.

**Comment by kevdouglass** [comment]
sounds good! Hey I have a PR coming in rn, do you have time to review it real quick this evening? I have more to add on to it but didn't want to make the PR to big

**Comment by melippmann** [comment]
@kevdouglass sure, send it through, if it isn't too giant I should be able to get through it tonight.
